2-projects/1-todo-list/script.js

```diff
@@ -1,6 +1,39 @@
 function populateTodoList(todos) {
   let list = document.getElementById("todo-list");
   // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+  todos.forEach((todo) => {
+    const newLi = document.createElement("li");
+    const newLiSpan = document.createElement("span");
+    const newCheckIcon = document.createElement("i");
+    const newTrashIcon = document.createElement("i");
+
+    (newCheckIcon.ariaHidden = "true"), (newTrashIcon.ariaHidden = "true");
+    newTrashIcon.classList.add("fa", "fa-trash");
+    newCheckIcon.classList.add("fa", "fa-check");
+    newLiSpan.classList.add("badge", "bg-primary", "rounded-pill");
+    newLi.classList.add(
+      "list-group-item",
+      "d-flex",
+      "justify-content-between",
+      "align-items-center"
+    );
+
+    newLi.textContent = todo.task;
+
+    newLiSpan.append(newCheckIcon, newTrashIcon);
+    newLi.appendChild(newLiSpan);
+    list.appendChild(newLi);
+
+    newCheckIcon.addEventListener("click", () => {
+      if (newLi.style.textDecoration === "none") {
+        newLi.style.textDecoration = "line-through";
+      } else {
+        newLi.style.textDecoration = "none";
+      }
+    });
+
+    newTrashIcon.addEventListener("click", () => list.removeChild(newLi));
```

Comment:
I like the way you grouped common/related code together like creating the elements and appending them and adding classes